internal/explorer/db/types.go

```diff
@@ -59,6 +61,8 @@ type Node struct {
 	RentContractID  int64
 	RentedByTwinID  int64
 	SerialNumber    string
+	Longitude       *float64
```

Comment:
I don't believe they should be pointers, and in the database that longitude and latitude columns shouldn't allow nulls 

I checked in the graphql https://graphql.grid.tf/graphql


```
query MyQuery {
  countries {
    id
    lat
    long
  }
}

```
Don't believe they should be nullables

Reply:
I also checked in the database scheme 
![image](https://user-images.githubusercontent.com/64129/209627568-518f7bd8-9065-4045-93d7-d45d8ddac8a4.png)

seems the columns aren't allowing null as well.

Reply:
Let me clarify my decisions. 

1. As you observed, long and lat are text types in the DB, and it looked ugly to me to return these supposedly decimal data from the API as it is(text), it would be required further conversion from the consumed clients. So I decide to return it as JSON numbers, and for this, I needed to convert the text to decimal.

3. I observed that there is some missing/invalid long, and lat data (not null, but empty text), that needs to be handled properly when casting to numbers, can't use an empty numeric value, 0, so I returned on purpose null in these cases as a more suitable choice.

4. The reason that I don't want to use an empty value for the numeric type, 0, to represent the missing/invalid entry is that 0 would consider a valid long, and lat.

5. This leads me to the decision of using *float64, to keep nullable values (after converting from text) from converted to empty float64 value 0 as struct gets filled.
To sum up it,
Original long, lat data are text types with some entries having empty text (missing/invalid)
API data: number type with some entries have null to represent missing/invalid values on original data.